automatic/ultravnc/ultravnc.nuspec

```diff
@@ -52,6 +52,10 @@ This is a nuspec. It mostly adheres to https://docs.nuget.org/create/Nuspec-Refe
     <description><![CDATA[
 UltraVNC is an open source application that uses the VNC protocol to control another computer remotely over a network connection.  UltraVNC allows the use of a remote computer as if the user were in front of it. This is achieved by sending mouse movements and key-presses to the remote computer, and replicating the remote computer's display (subject to differences in resolution) locally in real time. UltraVNC bears a strong resemblance to RealVNC Free Edition. However, in addition to remote control it adds various features, such as an encryption plugin to secure the client/server connection. It also supports file transfers, chat functionality and various authentication methods. The two computers must be able to communicate across a network, such as a local subnet, internal network, or the Internet.
 
+## Package Specific
+#### Package Parameters
+ * `/LoadInf:` - load an ultravnc inf file
+
```

Comment:
Could you please copy the changes in the description into the README.md file? The description element is filled from that file.

Could you also add a link to some documentation about this ini file?